dependencies.sh

```diff
@@ -1,3 +1,11 @@
 pip3 install --upgrade pip
-dnf install -y rust coreos-installer
+cat <<EOF | sudo tee /etc/yum.repos.d/kubernetes.repo
```

Comment:
It's unclear to me why this is a dependency for SR-IOV network operator? Because we run make deploy-setup?

Reply:
Yep